tests/Helper/RouteTest.php

```diff
@@ -28,6 +28,20 @@ public function testInvokeReturnsGeneratedRoute()
                   ]);
 
         $helper = $this->container->newRouteHelper();
-        $this->assertEquals('/blog/4%202/edit', $helper('test', ['id' => '4 2', 'foo' => 'bar']));
+        $this->assertEquals('/blog/42/edit', $helper('test', ['id' => '42', 'foo' => 'bar']));
```

Comment:
I think we should not modify these tests because these have some specific value.

Reply:
Possible bc break.
I test the value against token

> 
      if ($token) {
          if (!preg_match('~^' . $token . '$~', (string)$val)) {
              throw new \RuntimeException(sprintf(
                  'Parameter value for [%s] did not match the regex `%s`',
                  $name,
                  $token
              ));
          }
      }

Reply:
'([0-9]+)' don't match '4 2'

Reply:
I can delete it, no problem

Reply:
So are you saying 

```
$this->assertEquals('/blog/4%202/edit', $helper('test', ['id' => '4 2', 'foo' => 'bar']));
```

the above will work ?

Reply:
yes without adding specific token.

Reply:
Oh i see what you mean. Missed to look it clearly. This looks like a bug as you mentioned.

Reply:
I can commit the 2 tests and preserve value check